resources/js/Pages/Login.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="container-of-login ">
+    <div class="button-container">
+      <button v-if="!isLoggedIn" class="login-button" @click="toggleLoginForm">
+        {{ showLoginForm ? 'Hide Login' : 'Login' }}
+      </button>
+      <button v-if="isLoggedIn" class="logout-button" @click="logoutUser">
+        Logout
+      </button>
+    </div>
+    <div v-if="!isLoggedIn" class="login-form" :class="{ hidden: !showLoginForm }">
+      <form @submit.prevent="loginUser">
+        <div class="form-inside">
+          <label>Email</label>
+          <input type="email" v-model="user.email" class="text-inside-form" placeholder="Email">
+        </div>
+        <div class="form-inside">
+          <label>Password</label>
+          <input type="password" v-model="user.password" class="text-inside-form" placeholder="Password">
+        </div>
+        <button v-if="!isLoggedIn" type="submit" class="submit-login-button">Login</button>
+      </form>
+    </div>
+    <!--Component Register, which is visible when user is not logged in-->
+    <Register v-if="!isLoggedIn" :isLoggedIn="isLoggedIn" />
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+import Register from './Register.vue';
+
+export default {
+  name: 'Login',
+  components: { Register },
+  data() {
+    return {
+      result: {},
+      user: {
+        email: '',
+        password: ''
+      },
+      showLoginForm: false,
+      isLoggedIn: false
+    };
+  },
+  methods: {
+    toggleLoginForm() {
+      this.showLoginForm = !this.showLoginForm;
+    },
+    loginUser() {
+      axios
```

Comment:
We don't need axios, because we have Inertia.js in the project. Please refactor this.